src/main/java/UserText.java

```diff
@@ -0,0 +1,47 @@
+import java.util.ArrayList;
+import java.util.List;
+
+//Wrapper class for the list of tasks
+public class UserText {
```

Comment:
Maybe have the class be named as 'TaskList'? I was quite confused what 'UserText' would contain before reading the implementation.